misc/about_command.py

```diff
@@ -0,0 +1,52 @@
+import sublime
+import sublime_plugin
+import mdpopups
+from .. import __version__
+from ..api import get_version
+
+
+md_text = f"""
+# Pieces
+## Discover
+- [Plugins](https://code.pieces.app/plugins)
+
+## COMMUNITY
```

Comment:
this is all caps kind of randomly we should make it follow the rest of the titles